ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.1.13f1
-m_EditorVersionWithRevision: 2020.1.13f1 (5e24f28bfbc0)
+m_EditorVersion: 2019.4.20f1
```

Comment:
Hey you seem to be using the wrong version on Unity3d, try download version 2020.1.13f

Reply:
Is it okay to use 2020.3.0f1?